app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/SubscriptionServiceConnector.scala

```diff
@@ -35,26 +35,26 @@ class SubscriptionServiceConnector @Inject() (http: HttpClient, appConfig: AppCo
   ec: ExecutionContext
 ) {
 
-  private val url               = appConfig.getServiceUrl("subscribe")
-  private val loggerComponentId = "SubscriptionServiceConnector"
+  private val logger = Logger(this.getClass)
+  private val url    = appConfig.getServiceUrl("subscribe")
 
   def subscribe(request: SubscriptionRequest)(implicit hc: HeaderCarrier): Future[SubscriptionResponse] = {
-    val loggerId = s"[$loggerComponentId][subscribe] SUB02"
+    val loggerId = s"[subscribe] SUB02"
```

Comment:
Nit.  I guess this "prefix" is no longer required?  It looks like it was added to the log messages to identify the component the message came from.  But you have "fixed" that now.

Reply:
You mean to get rid of `subscription`?

Yeah, that's a good idea, because we will have name of the connector in the log, so it's not necessary